src/Components/AdvancedSearch/Internal/AdvancedSearchResultDetailsList/AdvancedSearchResultDetailsList.tsx

```diff
@@ -201,11 +199,37 @@ const AdvancedSearchResultDetailsList: React.FC<IAdvancedSearchResultDetailsList
         }
     };
 
+    const getAvailableProperties = () => {
+        let properties: string[] = [];
+        twins.forEach((twin) => {
+            properties = Object.keys(twin).concat(properties);
+        });
+        properties.sort((a, b) => (a.toLowerCase() > b.toLowerCase() ? 1 : -1));
+        return new Set(properties);
+    };
+    const updateColumns = (columnToAdd: string) => {
+        setSelectedColumnNames((currentValue) => {
+            return currentValue.concat(columnToAdd);
+        });
+    };
+    const deleteColumn = (columnToRemoveKey: string) => {
+        setSelectedColumnNames((currentValue) =>
+            currentValue.filter((col) => col !== columnToRemoveKey)
+        );
+    };
     return (
         <section className={classNames.root}>
-            <h3 className={classNames.listHeader}>
-                {t('advancedSearch.results', { twinCount })}
-            </h3>
+            <div className={classNames.listHeaderAndIcon}>
```

Comment:
IMO. If you search, add columns, then search again, it should include the searched columns in the list of results while maintaining the ones you added. I don't mind if it readds search columns that were removed (though it'd be nice if it remembered that).